README.md

```diff
@@ -1,6 +1,25 @@
+![LOGO](https://user-images.githubusercontent.com/62508697/139729943-d46fc1ee-9344-4563-9d7d-2369f38fce54.jpeg)
+
+
+
 # Falcon
```

Comment:
# Falcon 🦅 
**Easy – Fast – Done** :muscle:	

Our product is designed to help you manage your team's tasks and controls workflows in a better, easier way. It has an easy UI for managers and employees with a safe user login.
- Creating new tasks and assign them to your team.
- Overall view of all the tasks with smart filtering system.
- Comments section for discussion on each task.